### PR TITLE
[INLONG-2682][Manager] Add stream metric and stream config log related interface

### LINKED ISSUE #2682: [Feature][Manager] add metric and config log report interface
### Description

add metric and config log report interface

### Use case

add metric and config log report interface

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
add metric and config log report interface 

Fixes #2682

### CODE REVIEW

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <!-- Database connection URL, username, password -->
         <jdbcConnection driverClass="com.mysql.cj.jdbc.Driver"
                 connectionURL="jdbc:mysql://127.0.0.1:3306/apache_inlong_manager?nullCatalogMeansCurrent=true"
-                userId="root" password="inlong">
+                userId="root" password="12345678">
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/resources/generatorConfig.xml:41]: do not commit your env config.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    @Value("${msg.to.db.queue.size:10000}")
+    private int queueSize = 10000;
+
+    private volatile boolean isClose = false;
+
+    private LinkedBlockingQueue<T> dataQueue;
+
+    abstract boolean batchInsertEntities(List<T> entryList);
+
+    public boolean putData(T data) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java:45]: Add Javadoc for the public method.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    @Override
+    public void close() throws Exception {
+        isClose = true;
+    }
+
+    public void afterPropertiesSet() throws Exception {
+        dataQueue = new LinkedBlockingQueue(queueSize);
+        new Thread(() -> start(), "Batch_insert").start();
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java:59]: Its recommended to use Thread Pool, and give all threads a specified name.

Comment (baomingyu) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java (hunk 2 above)
Comment (baomingyu) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AbstractService.java:70]: done

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml (hunk 1 above)
Comment (baomingyu) [reply inlong-manager/manager-dao/src/main/resources/generatorConfig.xml]: done

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Review by ciscozhou — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**